src/handlers/graphical.rs

```diff
@@ -48,9 +50,16 @@ impl GraphicalReportHandler {
             theme,
             footer: None,
             context_lines: 1,
+            tab_width: None,
         }
     }
 
+    /// Replace tabs with spaces.
+    pub fn with_tab_width(mut self, width: usize) -> Self {
```

Comment:
You need to expose `.tab_width()` through `MietteHandler`, too, since that's the recommended way of customizing your handler in general.

Reply:
Done cb073dd3e773e5ac39db7d3ac811fcbf6c53e82d